fix(README): versionPrefix section (#31)
stackbuilders/semantic-release-hackage: README.md

[PVP](https://pvp.haskell.org/) is the standard versioning system for Haskell projects which is different from [Semantic Versioning](https://semver.org/). You may think that since this is a `semantic-release` plugin, you'll have to move out from PVP. But thanks to the `versionPrefix` option, you can keep the 4th digit in your version.

This is an optional feature enabling PVP compatibility when using this plugin.

> If your package cannot change the versioning system, `versionPrefix` can save the day and you can keep using PVP versioning together with this amazing tool.

Reviewer: This may still suggest people should move out of PVP. Let's try something a bit more neutral. How about:
```suggestion
So, if your package uses PVP, `versionPrefix` can save the day, and you can continue using PVP versioning with this amazing tool.
```

Also, I'd move this line along with the previous, as it continues the same idea 🙂

> However, using semantic versioning will simplify things and you will be following another well-known standard.

Reviewer: ```suggestion
Keep in mind that using Semantic Versioning does make things simpler while following another well-known standard.
```
